### PR TITLE
Example files fixes

### PR DESCRIPTION
**Pull request by mattiasostmar**
Related to three minor tasks:
Task: https://phabricator.wikimedia.org/T163316
https://phabricator.wikimedia.org/T163317
https://phabricator.wikimedia.org/T165056

### CODE REVIEW

File: create_infotexts.py [hunk 1]
```diff
@@ -195,6 +195,8 @@ def main():
         img = CypernImage()
         img.generate_list_of_stripped_keywords(metadata[fotonr]["Nyckelord"])
         img.create_commons_filename(metadata[fotonr])
+        img.add_cat_archaeological_exhibition_based_on_description(metadata[fotonr]["Beskrivning"])
```
Comment (lokal-profil) [inline create_infotexts.py]: Both lines are too long (>79 characters)

File: create_infotexts.py [hunk 2]
```diff
@@ -229,6 +231,32 @@ def __init__(self):
...
+        """
+        Special case fix to add content category in special cases from <Beskrivning>.
+
+        See task: https://phabricator.wikimedia.org/T163316
+
+        Populates self.content_cats
+
+        :return: None
+        """
+        if "interiör" in description and "grav" in description.lower():
```
Comment (lokal-profil) [inline create_infotexts.py]: I believe both of these be `.lower()`?

File: create_infotexts.py (hunk 1 above)
Comment (lokal-profil) [reply create_infotexts.py]: Would it make more sense to implement both these in the same function? `img.add_special_cats_based_on_descritpion(metadata[fotonr]["Beskrivning"])`

**Review by lokal-profil — CHANGES_REQUESTED** [review]

File: create_infotexts.py (hunk 2 above)
Comment (mattiasostmar) [reply create_infotexts.py]: Do they? With every word in description .lower() there can't be a match for "Interiör"?

File: create_infotexts.py (hunk 1 above)
Comment (mattiasostmar) [reply create_infotexts.py]: A) Done
B) I disagree. "Each function should do one thing and one thing only" and to understand special cases flow afterwards I think the naming of functions is more precise that way. OK?

File: create_infotexts.py (hunk 2 above)
Comment (lokal-profil) [reply create_infotexts.py]: To clarify your logic here:
`if ("interiör" in description) and ("grav" in description.lower())`
I.e. you check for "interiör" in `description` (not lower) but for "grav" in `description.lower()`

File: create_infotexts.py (hunk 1 above)
Comment (lokal-profil) [reply create_infotexts.py]: B) I still think checking for words in the description and adding a one-to-one category is "only one thing". I'll explain my reasoning below but I'm also ok with keeping it as it is.

Essentially the two functions you have are equivalent to a mapping which you are looping over
```python
words_to_cats = {
  "Archaeological_exhibitions": ("utställning", ),
  "Interiors_of_tombs": ("interiör", "grav)
}

for cat, words in words_to_cats.items():
    if all(word in description.lower() for word in words):
        self.content_cats.append(cat)
```
(note I haven't checked that the above code actually runs)

File: create_infotexts.py (hunk 2 above)
Comment (mattiasostmar) [reply create_infotexts.py]: Aha! Inconsistency! Done.

File: create_infotexts.py (hunk 1 above)
Comment (mattiasostmar) [reply create_infotexts.py]: Thank you for clarification, that helps! I leave it as is, but I will also see what happened with the keywords mappings, which might affect the final implementation.

File: create_infotexts.py [hunk 3]
```diff
@@ -192,11 +192,15 @@ def main():
...
+        keyw = metadata[fotonr]["Nyckelord"]
+        
         img = CypernImage()
-        img.generate_list_of_stripped_keywords(metadata[fotonr]["Nyckelord"])
+        
+        img.generate_list_of_stripped_keywords(keyw)
         img.create_commons_filename(metadata[fotonr])
-        img.add_cat_archaeological_exhibition_based_on_description(metadata[fotonr]["Beskrivning"])
-        img.add_cat_interior_of_tombs_based_on_description(metadata[fotonr]["Beskrivning"])
+        img.special_archaeological_exhibition_cat(desc)
```
Comment (lokal-profil) [inline create_infotexts.py]: Looks like you changed the name here but not to the actual functions

Comment (mattiasostmar) [reply create_infotexts.py]: Done.

**Comment by lokal-profil** [comment]
Ok to merge (on mobile so cannot click the approve button)

**Pull request merged**